### ISSUE TITLE
Fix accessibility issues identified by WAVE

### ISSUE THREAD

**Issue by anthkris** [hackid/hackid.github.io#9]
WAVE identified:

- Seach field needs a label
- Search button is empty
- Need ARIA roles on buttons
- Any way to fix redundant link issue? Do these need ARIA roles or do they act like links?
- Button blue and white fail contrast guidelines both ways (blue on white and white on blue)
